Modules/UpendoDnnSimpleAuthProvider/Components/Email.cs

```diff
@@ -88,7 +90,7 @@ public static void Send(string userEmail, string code)
     <tr>
       <td class=""header"" style=""padding: 24pt 0.75pt"">
         <div align=""center"">
-          <img src=""https://upendo-website.s3-us-west-1.amazonaws.com/Portals/0/Images/Upendo-logo-trans-230x65.png"" alt=""upendo logo"">
+         <img src=""http://"; emailBody += logo; emailBody += $@""" alt=""logo"">
```

Comment:
@ArielBlanco1990 We shouldn't be hard-coding `HTTP` / `HTTPS`.  You can default it to whatever the current request is using `Request.IsSecureConnection`.

Reply:
Understood!